packages/psd/src/interfaces/AdditionalLayerInfo/AliKey.ts

```diff
@@ -10,4 +10,24 @@ export enum AliKey {
   SectionDividerSetting = "lsct",
   TypeToolObjectSetting = "TySh",
   UnicodeLayerName = "luni",
+
+  VectorStrokeData = "vstk",
+  ObjectBasedEffects = "lfx2",
+  GradientFillSetting = "GdFl",
+  SolidColorSheetSetting = "SoCo",
+  PatternFillSetting = "PtFl",
+  VectorStrokeContentData = "vscg",
+  BlendOptionsCapacity = "iOpa",
+  VectorMaskSetting1 = "vmsk",
+  VectorMaskSetting2 = "vsms",
+  HueSaturation = "hue2",
+  NestedSectionDividerSetting = "lsdk",
+  LayerID = "lyid",
```

Comment:
Could you rename this to `LayerId`? We prefer to treat common acronyms and abbreviations as whole words when using CamelCase (e.g. HTML &rarr; Html). Ditto for `LayerIDAliBlock.ts` and `readLayerIDAliBlock.ts`.

Reply:
done :)